src/tests/tribol_mfem_common_plane.cpp

```diff
@@ -60,8 +60,9 @@ class MfemCommonPlaneTest : public testing::TestWithParam<std::pair<int, tribol:
     double lambda = 100000.0;
     // lame parameter (shear modulus)
     double mu = 100000.0;
-    // kinematic penalty
-    double p_kine = (lambda + 2.0 / 3.0 * mu) * std::pow(2.0, ref_levels - 1);
+    // kinematic constant penalty stiffness equivalent to the element-wise calculation, 
+    // which is bulk-modulus over element thickness.
+    double p_kine = (lambda + 2.0 / 3.0 * mu) / (1.0 / std::pow(2.0, ref_levels));
```

Comment:
Can you make the change in the corresponding example?  https://github.com/LLNL/Tribol/blob/develop/src/examples/mfem_common_plane.cpp

Reply:
Done.